examples/2d/texture_atlas.rs

```diff
@@ -50,8 +50,14 @@ fn setup(
 ) {
     let mut texture_atlas_builder = TextureAtlasBuilder::default();
     for handle in &rpg_sprite_handles.handles {
-        let texture = textures.get(handle).unwrap();
-        texture_atlas_builder.add_texture(handle.clone_weak().typed::<Image>(), texture);
+        let handle = handle.typed_weak();
+        let texture = match textures.get(&handle) {
+            Some(it) => it,
+            // If an asset in the folder happened to be valid, but not a texture,
+            // we don't want to add it to the atlas
+            None => continue,
```

Comment:
This is a behavioral change. I think we should keep unwrapping as `assets/textures/rpg` shouldn't contain any non-texture assets.

Reply:
Agreed here.